src/scipp/curve_fit.py

```diff
@@ -134,18 +134,11 @@ def _prepare_numpy_outputs(da, params, map_over):
 
 def _make_defaults(f, coords, params):
     spec = getfullargspec(f)
-    all_args = {*spec.args, *spec.kwonlyargs}
-    if not set(coords).issubset(all_args):
+    args = (set(spec.args) | set(spec.kwonlyargs)) - set(spec.kwonlydefaults or ())
```

Comment:
`scipp.scipy.optimize.curve_fit` requires keyword-only args for all but the first argument. Is `scipp.curve_fit` different? If not, `spec.args` can be removed here.

Reply:
Yes `scipp.curve_fit` is different in that regard.